.github/workflows/stale.yml

```diff
@@ -7,20 +7,23 @@ on:
 
 jobs:
   stale:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
Always better to use a specific version in my opinion. As the `latest` can keep changing.

Reply:
Yep I wanted it to change along with the use of `actions/stale@v3`, so we don't have to keep updating this.

Reply:
It's not about having to continuously update it. If it's a fixed version, we will always know up to which version it worked well with. @fzdy1914 your opinion.

Reply:
From my view, we should not use `an unspecified version` unless it is truly needed.

Reply:
I disagree in this context, but nonetheless I have reverted this change based on the consensus reached.

Reply:
@dcshzj It is to avoid unwanted bugs that may happen due to the latest version while the environment changes.